testserv.c

```diff
@@ -142,6 +166,12 @@ kill_srvpid(void)
 }
 
 #define SERVER() (progname=__func__, mustforksrv())
+#define SERVER_UNIX() ({                        \
+            progname=__func__;                  \
+            char *path = mustforksrv_unix();    \
+            assert(exist(path));                \
```

Comment:
Let's simplify this. We can move this assert into mustforksrv_unit() and then it can be made look similar to SERVER() macro.